rewrite-gradle/src/main/java/org/openrewrite/gradle/AddDependencyVisitor.java

```diff
@@ -333,6 +334,13 @@ public J.MethodInvocation visitMethodInvocation(J.MethodInvocation method, Execu
         }
     }
 
+    private String escapeIfNecessary(String configurationName) {
```

Comment:
In theory a gradle configuration could have a name with spaces and potentially escape characters but I chose not to attempt full string escaping.

Reply:
So we may need to be a little more explicit here rather than relying on the Groovy type names themselves. As an example, the `rewrite-gradle` module depends upon `rewrite-groovy` which presently uses the Groovy 3.x api surface. However, at runtime `rewrite-gradle` might be using Groovy 2.x or 3.x depending on what version of Gradle it is operating within.